plugins/pod-inspect.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pod-inspect
+spec:
+  version: v0.1.7
+  homepage: https://github.com/jpriebe/kubectl-pod-inspect
+  shortDescription: Get all of a pod's details at a glance
+  description: |
+    Provides detailed information about a pod, including its containers' statuses, pod events, and logs from non-ready containers.
```

Comment:
please wrap at 80 chars, looks better on user terminals that way.

Reply:
fixed.